### ISSUE TITLE
Erroring in chat

### ISSUE THREAD

**Issue by austin-wiechman** [CraftTweaker/CraftTweaker#418]
Errors shown in the minetweaker log show up in chat when you join a world. Since there are a lot of them, it is quite annoying.

#### Script used (Please Pastebin or gist your script, posting an unpasted or ungist'd script will automatically close this issue):
[https://pastebin.com/QheNKEnh](https://pastebin.com/QheNKEnh)

#### Minetweaker.log file (Please Pastebin or gist your file, posting an unpasted or ungist'd file will automatically close this issue):
[https://pastebin.com/MtaHw2sJ](https://pastebin.com/MtaHw2sJ)

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft: 1.12.2
- Forge: 14.23.1.2582
- Crafttweaker: 4.0.12

**Comment by kindlich** [comment]
Are you sure you linked the correct script?
The Erroring file is called `MineTweakerRecipeMaker.zs`

The Error you are getting is that you try to declare a variable that already has beed declared
e.g.
```
var myVariable = <ore:ingotIron>;
var myVariable = <ore:ingotIron>;
```

**Comment by austin-wiechman** [comment]
I renamed the file to script.zs and removed all the variables. This is the only file in my scripts folder.
New files:
script.zs: [https://pastebin.com/kUUEBfuT](https://pastebin.com/kUUEBfuT)
crafttweaker.log: [https://pastebin.com/dDTBhxvW](https://pastebin.com/dDTBhxvW)

**Comment by austin-wiechman** [comment]
After further research, I noticed that all the errors are from the same mod: Nuclearcraft. I went to report it to their issues page but found that someone else had already done so. It seems to be an issue between the latest version of CraftTweaker, and the recent versions of their mod.
[https://github.com/turbodiesel4598/NuclearCraft/issues/206](https://github.com/turbodiesel4598/NuclearCraft/issues/206)
